### PR TITLE
Prepare to migration to 1.0.0-RC

### PR DESCRIPTION
**Pull request by qwwdfsad**

### CODE REVIEW

File: ktor-http/ktor-http-cio/src/io/ktor/http/cio/internals/WeakTimeoutQueue.kt [hunk 1]
```diff
@@ -142,6 +142,7 @@ class WeakTimeoutQueue(
         }
     }
 
+    @UseExperimental(InternalCoroutinesApi::class)
     private class WeakTimeoutCoroutine<in T>(context: CoroutineContext, val delegate: Continuation<T>) :
```
Comment (qwwdfsad) [inline ktor-http/ktor-http-cio/src/io/ktor/http/cio/internals/WeakTimeoutQueue.kt]: `WeakTimeoutCoroutine` has probably the same bugs as `TimeoutCoroutine` had.
I've tried to comment some parts of cancellation machinery here and there, but none of `cio` tests have failed, so I decided not to touch this code.

File: ktor-server/ktor-server-servlet/src/io/ktor/server/servlet/ServletUpgrade.kt [hunk 2]
```diff
@@ -94,6 +94,8 @@ class ServletUpgradeHandler : HttpUpgradeHandler, CoroutineScope {
         launch(up.userContext, start = CoroutineStart.UNDISPATCHED) {
             val job = up.upgradeMessage.upgrade(inputChannel, outputChannel, up.engineContext, up.userContext)
 
+            // TODO it's parent-child?
```
Comment (qwwdfsad) [inline ktor-server/ktor-server-servlet/src/io/ktor/server/servlet/ServletUpgrade.kt]: Why are you not using attachChild API here?

Comment (cy6erGn0m) [reply ktor-server/ktor-server-servlet/src/io/ktor/server/servlet/ServletUpgrade.kt]: After discussion with Roman I'd remove it after beta release

File: ktor-http/ktor-http-cio/src/io/ktor/http/cio/internals/WeakTimeoutQueue.kt (hunk 1 above)
Comment (cy6erGn0m) [reply ktor-http/ktor-http-cio/src/io/ktor/http/cio/internals/WeakTimeoutQueue.kt]: Yes, this need to be updated to be in-sync with `TimeoutCoroutine`

**Pull request merged**